root/static/scripts/common/components/ReleaseEvents.js

```diff
@@ -123,7 +123,7 @@ const ReleaseEvents = ({
                   title={l('Show all release events')}
                 >
                   {bracketedText(texp.l('show {n} more', {
-                    n: events.length - COLLAPSE_THRESHOLD,
+                    n: events.length - 3,
```

Comment:
I'd still keep the constant though, just with the offset correction added:
```suggestion
                    n: events.length - COLLAPSE_THRESHOLD + 1,
```

Reply:
I didn't because it seemed arbitrary, but maybe? If anything, it'd feel better to have `events.length - (COLLAPSE_THRESHOLD - 2 + 1)`, since that's what we actually do. But I'll make the suggested change for now :)

Reply:
I can see what you mean. That might be more clear, particularly if we had a named constant for the `2` like `COLLAPSE_MIN` or something. I'd mainly just like that if we changed `COLLAPSE_THRESHOLD` and nothing else, the code would still work. :)